mifospay/src/main/java/org/mifos/mobilewallet/mifospay/savedcards/ui/CardsFragment.java

```diff
@@ -125,6 +129,7 @@ private void showEmptyStateView() {
     public void showErrorStateView(int drawable, int title, int subtitle) {
         rvCards.setVisibility(View.GONE);
         pbCards.setVisibility(View.GONE);
+        addCard.setVisibility(View.GONE);
```

Comment:
Since the data can be refreshed even when the user is on the current screen (due to swipe to refresh layout) and the cards may be populated then, make sure that the addCard chip is explicitly set to visible whenever the cards are populated.
This will prevent any inconsistent state in the UI.

Reply:
Sure, I'll check that